app/backplane/power_module/include/c_power_module.h

```diff
@@ -69,7 +69,7 @@ class CPowerModule : public CProjectConfiguration {
         "Broadcast Tenant", ipAddrStr, downlinkBroadcastPort, downlinkBroadcastPort, sensorDataDownlinkMessagePort
     };
     CDataLoggerTenant<NTypes::SensorData> dataLoggerTenant{
-        "Data Logger Tenant", "/lfs/sensor_data.bin", LogMode::Growing, 0, sensorDataLogMessagePort
+        "Data Logger Tenant", "/lfs/sensor_data.bin", LogMode::Growing, 0, sensorDataLogMessagePort, K_SECONDS(60), 5
```

Comment:
have you tested this? i feel like 60 seconds is a really long time to do syncing (and you'll end up losing data in the mean time)
Is the 60 seconds necessary? and like how much is necessary. in your testing did it need all 60 seconds?

Reply:
Well the 5 is there for every 5 measurements since we'll log constantly when flying. 60 seconds is for when we're logging on the pad every minute.

Reply:
And yes this was tested
<img width="1128" alt="Screenshot_20250511_154042" src="https://github.com/user-attachments/assets/20f807c4-03d3-4681-a4dc-6b81da3f5f33" />

Reply:
Forgot to attach the SS in the pr mb

Reply:
ohhhh wait yeah I was being dumb